ui/e2e/src/pages/PanelGroup.ts

```diff
@@ -103,4 +109,34 @@ export class PanelGroup {
   async addPanel() {
     await this.addPanelButton.click();
   }
+
+  /**
+   * Get information about the bounds of the panel group.
+   */
+  async getBounds() {
+    const groupBounds = await this.container.boundingBox();
+
+    // These values shouldn't be null in the cases we are using them, so thowing an error
+    // in the rare care they are null. This appropriately fails the test and acts as a
+    // type guard to simplify using the bounds in tests.
+    if (!groupBounds) {
+      throw new Error(`Unable to get bounds for panel group.`);
+    }
+
+    return groupBounds;
+  }
+
+  /**
+   * Gets the percentage of the height and width of a panel within the group.
+   * Useful for asserting resizing and responsive behavior.
+   */
+  async getPanelPercentOfBounds(panel: Panel) {
```

Comment:
I was struggling with naming this helper. Let me know if you have better ideas.

Reply:
Something like getPanelSizeRatio or getPanelBoundaryRatio comes to mind, but not sure that's better 🤔